src/plugins/UserStats.js

```diff
@@ -13,23 +13,41 @@ export default class UserStats extends Plugin {
         };
     }
 
+    wordCount(text) {
+        let words = text.trim().replace(/\s+/gi, ' ').split(' ');
+        if (words[0])
```

Comment:
This is always true, unless `text` is undefined or something. Also, you don't need the `+` in the regex, since you already have the `g` flag.

```js
return text.trim().replace(/\s/gi, ' ').split(' ').length;
```